modules/cluster-and-ha-management/pages/region-aware.adoc

```diff
@@ -0,0 +1,226 @@
+= Region Awareness
+//:page-aliases: tigergraph-server:region-aware:region-aware.adoc, tigergraph-server:region-aware:index.adoc
+:description: Overview of region awareness for TigerGraph servers.
+
+Region awareness represents a more sophisticated High Availability (HA) configuration within TigerGraph servers. In contrast to xref:ha-overview.adoc[High Availability], region-aware clusters offer HA at the **region/zone level** rather than the **individual node level**.
```

Comment:
```suggestion
Region awareness represents a more sophisticated High Availability (HA) configuration within TigerGraph servers. In contrast to standard xref:ha-overview.adoc[High Availability] clusters, region-aware clusters offer HA at the **region/zone level** rather than the **individual node level**.
```